spec/features/admin/admin_users_spec.rb

```diff
@@ -93,14 +95,14 @@
       end
 
       it "should show page with  new data" do
-        page.should have_content("bigbang@mail.com")
-        page.should have_content("Big Bang")
+        expect(page).to have_content("bigbang@mail.com")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.